contracts/hardhat.config.ts

```diff
@@ -46,25 +46,29 @@ const config: HardhatUserConfig = {
         accountPath: '~/.starknet_accounts',
       },
     },
+    requestTimeout: 1000000,
   },
   networks: {
     devnet: {
       url: 'http://127.0.0.1:5050',
+      args: ['--cairo-compiler-manifest', '../vendor/cairo/Cargo.toml'],
     },
     integratedDevnet: {
       url: 'http://127.0.0.1:5050',
       venv: 'active',
-      args: ['--lite-mode'],
+      args: ['--lite-mode', '--cairo-compiler-manifest', '../vendor/cairo/Cargo.toml'],
```

Comment:
We should pull in the latest https://github.com/0xSpaceShard/starknet-hardhat-plugin/releases/tag/v0.8.0-alpha.1 hardhat plugin because it defaults to cairo-lang 0.11.0.2. They removed manifestPath option and replaced it with cairo1BinDir